src/elements/content-explorer/ContentExplorer.tsx

```diff
@@ -1635,7 +1635,7 @@ class ContentExplorer extends Component<ContentExplorerProps, State> {
         const isDefaultViewMetadata: boolean = defaultView === DEFAULT_VIEW_METADATA;
         const isErrorView: boolean = view === VIEW_ERROR;
 
-        const viewMode = this.getViewMode();
+        const viewMode = isDefaultViewMetadata ? VIEW_METADATA : this.getViewMode();
```

Comment:
what's this change? viewMode means list or grid modes, not metadata vs recents vs search, etc

Reply:
I think you're thinking of the `view` rather than `viewMode`?